src/view/frontend/templates/translation.phtml

```diff
@@ -0,0 +1,100 @@
+<?php
+use Magento\Framework\Escaper;
+
+/** @var Escaper $escaper */
+
+$stringsToTranslate = [
+    "-- Please Select --",
+    "%1 zipcode: %1",
+    "%1 is required",
+    "Actions",
+    "Address update failed. Please try again.",
+    "Bill Here",
+    "Billing address made same as shipping address",
+    "Billing address update failed. Please try again.",
+    "Billing address updated successfully.",
+    "BILLING ADDRESS",
+    "Billing Information",
+    "Cancel",
+    "Cart Subtotal",
+    "Cart updated successfully.",
+    "Checkout Agreements",
+    "CHOOSE FROM THE ADDRESS LIST",
+    "City",
+    "Company",
+    "Country",
+    "Edit",
+    "Email address is saved.",
+    "Email address update failed. Please try again.",
+    "Email is invalid",
+    "Email is required",
+    "Email",
+    "First name",
+    "FROM ADDRESS BOOK",
+    "I agree",
+    "I will sign-in and checkout",
+    "Item",
+    "Last name",
+    "Login failed.",
+    "MOST RECENTLY USED",
+    "New Address",
+    "NEW ADDRESS",
+    "No cart items available",
+    "No payment methods available at the moment",
+    "No shipping methods available at the moment",
+    "Order Summary",
+    "Order Total",
+    "OTHER ADDRESSES",
+    "Password is required",
+    "Password",
+    "Payment method added successfully.",
+    "Payment method not available",
+    "Payment Methods",
+    "phone: %1",
+    "Phone",
+    "Place Order",
+    "Please agree with the terms & conditions",
+    "Please provide your billing address information.",
+    "Please provide your email address.",
+    "Please provide your login details.",
+    "Please provide your shipping address information.",
+    "Please select your payment method.",
+    "Please select your shipping method.",
+    "Postal Code",
+    "Price: %1",
+    "Price",
+    "Product Details",
+    "Qty",
+    "REMOVE",
+    "Required",
+    "Save",
+    "SELECTED ADDRESS",
+    "Selected shipping method is not available due to the shipping address change. Please select from the available methods.",
+    "Ship Here",
+    "Shipping address update failed. Please try again",
+    "Shipping address updated successfully.",
+    "SHIPPING ADDRESS",
+    "Shipping Information",
+    "Shipping method not available",
+    "Shipping method updated successfully.",
+    "Shipping Methods",
+    "Shipping",
+    "Sign In",
+    "Something went wrong while adding the payment method to the quote.",
+    "Something went wrong while updating shipping method",
+    "Something went wrong while updating the cart item.",
+    "State",
+    "Street",
+    "Total",
+    "Update",
+    "Use this address as my billing address",
+    "Your checkout details are not valid. Please verify your details."
+];
+$translatedStrings = [];
```

Comment:
So the only change we discussed here was for additional translation strings to be passed here through block arguments, using layout XML (theoretically it could be done in other ways too).
I'd suggest moving the strings above to a xml file as well